src/js/components/Button/doc.js

```diff
@@ -77,40 +65,57 @@ with plain Buttons.`,
         `The amount of spacing between icon and label in the button.`,
       )
       .defaultValue('small'),
+    hoverIndicator: hoverIndicatorPropType
+      .description(
+        `The hover indicator to apply when the user is mousing over the
+button. An object can be also be specified for color index support:
+{background: 'neutral-2'}. This prop is meant to be used only
+with plain Buttons.`,
+      )
+      .defaultValue(false),
+    href: PropTypes.string.description(
+      'If specified, the button will behave like an anchor tag.',
+    ),
+    icon: PropTypes.element.description('Icon element to place in the button.'),
     label: PropTypes.node.description('Label text to place in the button.'),
     onClick: PropTypes.func.description(
       `Click handler. Not setting this property and not specifying a href
-causes the Button to be disabled.`,
+        causes the Button to be disabled.`,
     ),
     plain: PropTypes.bool
       .description(
         `Whether this is a plain button with no border or pad.
-Non plain button will show both pad and border.
-The plain button has no border and unless the icon prop exist it has no pad as 
-well.`,
+          Non plain button will show both pad and border.
+          The plain button has no border and unless the icon prop exist it has 
+          no pad as well.`,
       )
       .defaultValue(false),
     primary: PropTypes.bool
       .description(
         `Whether this is a primary button. There should be at most one per page
-         or screen.`,
+            or screen.`,
       )
       .defaultValue(false),
     reverse: PropTypes.bool
       .description(
         `Whether an icon and label should be reversed so that the icon is at the
-end of the anchor.`,
+              end of the anchor.`,
       )
       .defaultValue(false),
+    size: PropTypes.oneOf(['small', 'medium', 'large']).description(
```

Comment:
This is the only section that was added to this file. 
The other diffs are organization changes.